### PR TITLE
Added back to top button and improve load time.

### LINKED ISSUE #4: Add "Back to top" button for mobile view

### PR DESCRIPTION
**Pull request by DanielSokil**
Fixes #4

### CODE REVIEW

File: src/views/layouts/layout.slang [hunk 1]
```diff
@@ -16,30 +16,38 @@ html lang="en"
...
-    link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/styles/github.min.css"
+    link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/styles/github.min.css"
     link rel="stylesheet" href="/css/crystal-shards.css"
 
     title #{Config.config.page_title}
     link href="/images/favicon.ico" rel="shortcut icon" type="image/x-icon"
   body
     == render "src/views/partials/navbar.slang"
     == render "src/views/partials/about.slang"
+    == render "src/views/partials/back_to_top.slang"
```
Comment (mamantoha) [inline src/views/layouts/layout.slang]: src/views/partials/back_to_top.slang doesn't exist

Comment (DanielSokil) [reply src/views/layouts/layout.slang]: Silly mistake (:

File: public/css/crystal-shards.css [hunk 2]
```diff
@@ -241,3 +241,40 @@ ul.not-list li {
...
+  position: fixed;
+  bottom: 16px;
+  right: 16px;
+  z-index: 9999;
+  width: 32px;
+  height: 32px;
+  text-align: center;
+  line-height: 30px;
+  background: rgba(0, 0, 0, 0.6);
+  color: rgba(255, 255, 255, 0.6);;
```
Comment (mamantoha) [inline public/css/crystal-shards.css]: redundant `;`

Comment (DanielSokil) [reply public/css/crystal-shards.css]: I need a style linter.

**Pull request merged**